src/messages/models/Banner/validateOptions.js

```diff
@@ -38,9 +39,18 @@ function getValidVal(logger, typeArr, val, location) {
     if (validateType(type, val)) {
         if (type === Types.STRING && validVals.length > 0) {
             // Check if aliased value used.
-            const validVal = arrayFind(validVals, v => v.split('|').some(x => x === val));
+            const validVal = arrayFind(validVals, v => {
+                if (typeof v !== 'string') {
+                    return false;
+                }
+                return v.split('|').some(x => x === val);
+            });
+
             if (validVal === undefined) {
                 logInvalidOption(logger, location, validVals, val);
+                if (validVals[0] === undefined) {
```

Comment:
Consider `typeof validVals[0] === 'undefined'` instead when checking for `undefined`